pkg/sysinfo/sysinfo.go

```diff
@@ -102,6 +117,128 @@ type cgroupPids struct {
 	PidsLimit bool
 }
 
+// ValidateHugetlb check whether hugetlb pagesize and limit legal
+func (c cgroupHugetlbInfo) ValidateHugetlb(pageSize string, limit uint64) (string, []string, error) {
+	var (
+		w   []string
+		err error
+	)
+	if pageSize != "" {
+		if err = isHugepageSizeValid(pageSize); err != nil {
+			return "", w, err
+		}
+	} else {
```

Comment:
I see the proposed format for the CLI is `--hugetlb-limit 2m:6m` - in what case would `pageSize` be empty? (the flag format may need some discussion altogether)

Reply:
I'd rather `--hugetlb-limit 6M` to be legal option, because `docker info` will display the default hugepage size(2MB .e.g), in such case `pagesize` is set to the default one, and 6M will be taken as limit.